style/themes/lcars.css

```diff
@@ -1361,44 +1361,6 @@ table.table-bordered.dataTable td {
   border: 1px solid #111 !important;
 }
 
-table.dataTable thead .sorting::before {
-  content: "\25B3";
-  position: absolute;
-  bottom: 16px;
-  right: 8px;
-  display: block;
-  font-family: "Glyphicons Halflings";
-  opacity: 0.3;
-  font-size: 0.6em;
-}
-
-table.dataTable thead .sorting:after,
-table.dataTable thead .sorting_asc:after,
-table.dataTable thead .sorting_desc:after,
-table.dataTable thead .sorting_asc_disabled:after,
-table.dataTable thead .sorting_desc_disabled:after {
-  opacity: 0.6;
-}
-
-table.dataTable thead .sorting:after {
-  opacity: 0.3;
-  content: "\25BD";
-  font-size: 0.6em;
-}
-
-table.dataTable thead .sorting_asc::after {
-  content: "\25B2";
-}
-
-table.dataTable thead .sorting_desc::after {
-  content: "\25BC";
-}
-
-table.dataTable thead .sorting_asc_disabled:after,
-table.dataTable thead .sorting_desc_disabled:after {
-  color: #eee;
-}
-
 .table-striped > tbody > tr:nth-of-type(2n + 1) {
```

Comment:
The new CSS changed the selector here, so we need to changed it to make sure the table rows have alternating background colors:

```suggestion
table.dataTable.table-striped > tbody > tr:nth-of-type(2n + 1) {
```

Reply:
What about using `tr.odd` instead of `tr:nth-of-type(2n + 1)`?
```suggestion
table.dataTable.table-striped > tbody > tr.odd {
```
From the Domains page:
```html
<tbody>
    <tr
        class="odd"
        data-id="0064006f006d00610069006e0031002e0063006f006d_deny_exact"
    ></tr>
    <tr
        class="even"
        data-id="0064006f006d00610069006e0032002e0063006f006d_deny_exact"
    ></tr>
    <tr
        class="odd"
        data-id="0064006f006d00610069006e0033002e0063006f006d_deny_exact"
    ></tr>
    <tr
        class="even"
        data-id="0064006f006d00610069006e0034002e0063006f006d_deny_exact"
    ></tr>
</tbody>
```

Reply:
Both `table.dataTable.table-striped > tbody > tr.odd` and `table.dataTable.table-striped > tbody > tr:nth-of-type(2n + 1)` seem to work for Domains, Groups, Clients, Lists and Query Log.
But neither seems to work for the "homepage":
Example using Midnight:
This is with `table.dataTable.table-striped > tbody > tr.odd` and `table.dataTable.table-striped > tbody > tr:nth-of-type(2n + 1)`:
<img width="1002" height="494" alt="Screenshot from 2025-11-23 10-56-19" src="https://github.com/user-attachments/assets/701ad54b-d926-42f4-9d3d-3e4e656bfd92" />
<img width="1002" height="494" alt="Screenshot from 2025-11-23 10-59-13" src="https://github.com/user-attachments/assets/5aa4e806-8245-4940-9794-fc35e268391a" />

The old `.table-striped > tbody > tr:nth-of-type(2n + 1)` is working for the homepage, but not something like Domains:
<img width="1002" height="494" alt="Screenshot from 2025-11-23 10-58-02" src="https://github.com/user-attachments/assets/7f549364-b59f-4358-abe3-4c27b40d6f87" />
<img width="1002" height="494" alt="Screenshot from 2025-11-23 10-58-24" src="https://github.com/user-attachments/assets/9fa79c89-c083-4e60-93a5-f946e25370ca" />

The table on the homepage does not have `even` and `odd` classes.

We would need something like this for default-dark.css:
```css
.table-striped > tbody > tr:nth-of-type(2n + 1) {
  background-color: #2d343a;
}
table.dataTable.table-striped > tbody > tr.odd {
  background-color: #2d343a;
}
```

The same is true for all other themes as well.
For LCARS:
```suggestion
table.dataTable.table-striped > tbody > tr.odd {
  background: none;
  background-color: rgba(80, 80, 80, 0.1);
}
.table-striped > tbody > tr:nth-of-type(2n + 1) {
```

Reply:
For High contrast light we dont seem to need it at all.
With
```css
.table-striped > tbody > tr:nth-of-type(2n + 1) {
  background-color: #fcfcfc;
}
table.dataTable.table-striped > tbody > tr.odd {
  background-color: #fcfcfc;
}
```
<img width="1002" height="494" alt="Screenshot from 2025-11-23 11-10-11" src="https://github.com/user-attachments/assets/02f28b3f-962b-41a5-a9d8-f8a743166ac3" />
<img width="1002" height="494" alt="Screenshot from 2025-11-23 11-10-36" src="https://github.com/user-attachments/assets/1a81e072-7f27-4283-8736-4ee56af1749e" />

Without:
<img width="1002" height="494" alt="Screenshot from 2025-11-23 11-12-10" src="https://github.com/user-attachments/assets/c0de0dee-68a8-4e6c-a9e0-98dbec752151" />
<img width="1002" height="494" alt="Screenshot from 2025-11-23 11-12-34" src="https://github.com/user-attachments/assets/726fbd27-c007-4b22-9899-e1dcfdef1ab3" />

We might want to keep it, but use a color with better contrast instead, I will commit the Without changes for now.

Reply:
Hmmm...

I only used the group management pages for testing my suggestions.

Your suggestions will probably fix all issues, but I will take a look an check (using the last commit) why some tables are working differently, to make sure nothing else is broken.